### ISSUE TITLE
deploy --local-only still uses remote builder

### ISSUE THREAD

**Issue by dcecile** [superfly/flyctl#374]
Based on #237, the `--local-only` flag should prevent using a remote builder, right?

But on my setup (Ubuntu desktop, not inside Docker or CI) a remote builder still gets picked:

```shell
$ flyctl version  
flyctl v0.0.188 linux/amd64 Commit: d859925 BuildDate: 2021-03-11T22:40:39Z

$ flyctl deploy --local-only
Deploying throbbing-dust-8272
==> Validating App Configuration
--> Validating App Configuration done
Services
TCP 80/443 ⇢ 8080 

Deploy source directory '/home/dcecile/Code/fertile-earth'
==> Building with Dockerfile
Using Dockerfile Builder: /home/dcecile/Code/fertile-earth/Dockerfile
INFO Local docker unavailable, hooking you up with a remote Docker builder...
INFO Waiting for remote builder (fly-builder-sparkling-bird-702) to become available...
```

**Comment by michaeldwan** [comment]
This is fixed as of v0.0.197+

**Comment by dcecile** [comment]
Thanks!
